Mathlib/MeasureTheory/Function/LpSeminorm/ChebyshevMarkov.lean

```diff
@@ -15,58 +15,102 @@ open scoped NNReal ENNReal
 
 namespace MeasureTheory
 
-variable {α E : Type*} {m0 : MeasurableSpace α} [NormedAddCommGroup E]
-  {p : ℝ≥0∞} (μ : Measure α) {f : α → E}
+variable {α E ε' : Type*} {m0 : MeasurableSpace α} [NormedAddCommGroup E]
+  [TopologicalSpace ε'] [ContinuousENorm ε']
+  {p : ℝ≥0∞} (μ : Measure α)
 
-theorem pow_mul_meas_ge_le_eLpNorm (hp_ne_zero : p ≠ 0) (hp_ne_top : p ≠ ∞)
-    (hf : AEStronglyMeasurable f μ) (ε : ℝ≥0∞) :
-    (ε * μ { x | ε ≤ (‖f x‖₊ : ℝ≥0∞) ^ p.toReal }) ^ (1 / p.toReal) ≤ eLpNorm f p μ := by
+theorem pow_mul_meas_ge_le_eLpNorm_enorm (hp_ne_zero : p ≠ 0) (hp_ne_top : p ≠ ∞)
```

Comment:
Up to definitional equality a direct generalization of the lemma below, right? I'd suggest just replace the lemma below with this. In other PRs (when we just had the ENorm class) we also just replaced statements involving `(‖y‖₊ : ℝ≥0∞)` by `‖y‖ₑ`

Also for the results below.